Mobile/iOS/Templates/Release_3.4.9/xxx-thumbnails_not_downloaded.md

```diff
@@ -0,0 +1,48 @@
+|16|Rename file (conflict)| 1. Upload an image<br>2. Upload another image with the same name in the folder<br>3. Solve the conflict with "rename"|1. Thumbnail is shown <br>2. A conflict is detected<br>3. Both thumbnails are shown |||
+|17|Delete file locally| 1. Upload an image<br>2. Download the image<br>3. Delete locally the image|1. Thumbnail is shown <br>2. Thumbnail is shown <br>3. Thumbnail is shown |||
+|18|Share file|1. Upload an image<br>2. Share the image to another user<br>3. Login with this user|1. Thumbnail is shown <br>3. Thumbnail is shown for the sharee|||
+|19|Favourite file|1. Set an image as favourite|Thumbnail is shown |||
+|20|Update favourite|1. Set an image as favourite<br>2. From the web, upload another one with the same name overwriting the first one|The thumbnail in device is updated|||
+|21|Favourite folder|1. Create a new folder and upload a large amount of images<br>2. Set the folder as favourite|All thumbnails are shown|||
+|22|Cancel Download|1. Create a new folder and upload a large amount of images<br>2. Swipe to download the folder<br>3. Cancel the downloading| Thumbnails are shown independent whether the image was downloaded or not|||
+|23|Cancel Favourite|1. Create a new folder and upload a large amount of images<br>2. Swipe to set the folder as favourite<br>3. Cancel the favourite| Thumbnails are shown independent whether the image was downloaded or not|||
+|**Offline**||||||
+|24|Offline |1. In web, create an account and upload some images/videos in a folder that is not root<br>2. In app, login in the account but not browse<br>3. Switch the device network connection off<br>4. Browse into the folder<br>5. Switch the device network connection on and browse|4. Thumbnails are not shown<br>5. Thumbnails are correctly shown|||
+|**Multiaccount**||||||
+|25|Multiaccount| 1. Upload some images in different accounts and change from one to another|Thumbnails are correctly shown in all accounts|||
+|26|Multiaccount| 1. Upload  some image files in a folder call "folder1"<br>2. Change to account2<br> 3.Upload some image files in a folder call "folder1"  <br>4. Change the account to the first one|1. Thumbnails of account 1 are visible correctly<br>3. Thumbnails of account 2 are visible correctly<br>4. Thumbnails of account 1 keep on being correct|||
+|**Shibboleth**||||||
+|27|Shibb server|1. Login in a Shibboleth server<br>2. Upload some images<br>3. Download some images|2. Thumbnails are correctly shown<br>3. Thumbnails are correctly shown|||
+|**Cache**||||||
+|28|Cache|1. Upload some images to different accounts<br>2. Move some images from a location to another one<br>3. Delete some images<br>4. Delete an account | Check in all the cases that the thumbnails are correctly managed in mobile's cache|||
```

Comment:
Changes on DB made, we need a test case to check update app from an older version 3.4.8